hummingbot/wallet/ethereum/web3_wallet_backend.py

```diff
@@ -296,11 +303,12 @@ def stop(self):
             self._check_transaction_receipts_task = None
 
     async def check_network(self) -> NetworkStatus:
+        # Assume connected if received new blocks in last 2 minutes
+        if time.time() - self._last_timestamp_received_blocks > 60 * 2:
+            return NetworkStatus.CONNECTED
+
         try:
-            await self._ev_loop.run_in_executor(hummingbot.get_executor(),
-                                                getattr,
-                                                self._w3.eth,
-                                                "blockNumber")
+            await self._update_gas_price()
```

Comment:
This is not needed - we're already updating gas prices at new block.

Reply:
yea it's not needed but might as well do a task that is useful? rather than just checking blockNumber and ignoring result.